src/masternode-sync.h

```diff
@@ -24,6 +29,11 @@
 class CMasternodeSync;
 extern CMasternodeSync masternodeSync;
 
+struct TierTwoPeerData {
+    // map of message --> last request timestamp, bool hasResponseArrived.
+    std::map<const char*, std::pair<int64_t, bool>> mapMsgData;
+};
```

Comment:
instead of a struct with a single variable, it would be cleaner with a typedef.
This way all `peerData.mapMsgData` would be just `peerData`.

Reply:
I left it as a struct because will add other members moving forward with the new syncing process. This is a naive regtest-only sync.
It basically will mean less changes in the upcoming work.